JSDemos/Demos/Lookup/Basics/description.md

```diff
@@ -1 +1,20 @@
-The Lookup is&nbsp;a&nbsp;UI component that allows an&nbsp;end user to&nbsp;search for an&nbsp;item in&nbsp;a&nbsp;collection shown in&nbsp;a&nbsp;drop-down menu. This demo shows how to&nbsp;create a&nbsp;simple and a&nbsp;grouped Lookup.
\ No newline at end of file
+The Lookup allows users to search through its drop-down list and select a single item from it. The drop-down list can be simple or grouped by a single category.
+
+### Use a simple drop-down list
+To bind the Lookup to a simple list, use the [items[]](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/items/) property. The data array should include strings only.
+
+### Use a grouped drop-down list
+To create a grouped list, do the following:
+
+* Use the [dataSource](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#dataSource) property to specify a DataSource object. Set this object's [store](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#store) property to a data array and the [group](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#group) property to a category field.
+
+* Use the [displayExpr](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#displayExpr) property to specify the data field whose values should be displayed.
+
+* Set the [grouped](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#grouped) property to **true**.
+
+You can also use a pregrouped data array. Each object of this array should include the `key` (category) and `items` (drop-down values) fields. In this case, do not specify the DataSource.**group** and **displayExpr** properties.
+
+### Configure the drop-down list
+The Popover component represents the drop-down list. To customize it, specify any Popover's properties inside the [dropDownOptions](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#dropDownOptions) property. In this demo, we disabled the [showTitle](/Documentation/ApiReference/UI_Components/dxPopover/Configuration/#showTitle) property to hide Popover's title and enabled the [closeOnOutsideClick](/Documentation/ApiReference/UI_Components/dxPopover/Configuration/#closeOnOutsideClick) property for the Grouped Lookup.
```

Comment:
Popup can be used instead of Popover if we set `usePopover` property to `false`

Reply:
First, let's add a link to the Popover

>To customize it, specify any Popover's properties inside the [dropDownOptions](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#dropDownOptions) property.

It is not a good idea to use such prepositions as 'in' or 'inside'. [Review this thread please](https://teams.microsoft.com/l/message/19:28d4b0475af64e44a48057a5b9cb1fd9@thread.skype/1616007813544?tenantId=e4d60396-9352-4ae8-b84c-e69244584fa4&groupId=9298adba-77e3-4b3e-a0eb-b3c426b4ba6d&parentMessageId=1616007813544&teamName=DX%20Tech%20Writers%20Club&channelName=General&createdTime=1616007813544)

>In this demo, we disabled the [showTitle](/Documentation/ApiReference/UI_Components/dxPopover/Configuration/#showTitle) property to hide Popover's title and enabled the [closeOnOutsideClick](/Documentation/ApiReference/UI_Components/dxPopover/Configuration/#closeOnOutsideClick) property for the Grouped Lookup.

These two cases seem unrelated to each other.